ndb_api/common_helpers.go

```diff
@@ -45,3 +46,8 @@ func GetDatabasePortByType(dbType string) int32 {
 		return -1
 	}
 }
+
+func ValidateDatabaseProvisionRequest(req DatabaseProvisionRequest) error {
+	err := validator.New().Struct(req)
+	return err
+}
```

Comment:
We might not need this at all when we move the validation into ndb_api.ProvisionDatabase function..